devservices/config.yml

```diff
@@ -0,0 +1,47 @@
+x-sentry-service-config:
+  version: 0.1
+  service_name: uptime-checker
+  dependencies:
+    kafka:
+      description: Shared instance of kafka used by sentry services
+      remote:
+        repo_name: sentry-shared-kafka
+        branch: main
+        repo_link: git@github.com:getsentry/sentry-shared-kafka.git
+    redis:
+      description: Shared instance of redis used by sentry services
+      remote:
+        repo_name: sentry-shared-redis
+        branch: main
+        repo_link: git@github.com:getsentry/sentry-shared-redis.git
+    uptime-checker:
+      description: Service powering Sentry's uptime detection features
+  modes:
+    default: [redis]
```

Comment:
should kafka be in default mode?

Reply:
We don’t produce anything to Kafka in tests